### PR TITLE
fix: Use now instead of ARR/BRD for bus

### PR DESCRIPTION
**Pull request by BrandonTR**
### Summary

_Ticket:_ [For bus, use Now instead of ARR/BRD](https://app.asana.com/0/1205732265579288/1207678190799457/f)

For bus, don't show `ARR` or `BRD` if the trip is less than or equal to 30 seconds away,  instead show `Now`. Otherwise round to nearest minute.

### Testing
Added some unit tests, tweaked existing tests, and added a UI test for Android.

### CODE REVIEW

File: iosApp/iosApp/ComponentViews/UpcomingTripView.swift [hunk 1]
```diff
@@ -68,6 +68,8 @@ struct UpcomingTripView: View {
             case .hidden, .skipped:
                 // should have been filtered out already
                 Text(verbatim: "")
+            case .now:
+                Text("Now").font(Typography.headlineBold)
```
[the remark this answers is not in the document]
Comment (JackVCurtis) [reply iosApp/iosApp/ComponentViews/UpcomingTripView.swift:72]: Good question, I think this is worth raising in Slack. Also, am I missing something or are these strings not internationalized? If so can we address that while we're in here?

[the remark this answers is not in the document]
Comment (BrandonTR) [reply iosApp/iosApp/ComponentViews/UpcomingTripView.swift:72]: On iOS it works a little differently where strings that appear hard coded are actually put into a strings catalog, essentially the english translation becomes the key for the translated values. Also good point, I'll raise this in the team channel

**Review by KaylaBrady — APPROVED** [review]
Nice!

**Review by JackVCurtis — APPROVED** [review]
Looks good!

**Pull request merged**